about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styleabout.css" />
+    <link rel="stylesheet" href="https://use.typekit.net/uzf0rar.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>About</title>
+</head>
+<body>
+    <section class="aboutbackground">
+        <nav class ="desktopnav">
+            <div class="space"></div>
+            <ul class="socialmedia">
+                <li><a href=""><i class=" fa fa-brands fa-facebook-f"></i></a></li>
+                <li><a href=""><i class=" fa fa-brands fa-twitter"></i></a></li>
+                <li><a href="">English</a></li>
+                <li><a href="">My Page</a></li>
+                <li><a href="">Logout</a></li>
+            </ul>
+        </nav>
+        <nav class ="desktop2nav">
+            <div class="space2"><img src="image/logo-EHS.png" class="desktoplogo"></div>
+            <ul class="socialmedia2">
+                <li><a href="index.html" id="about">Home</a></li>
+                <li><a href="">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href="" id="Campaign">CC Campaign</a></li>
+            </ul>
+        </nav>
+        <div class="meuncomponents">
+            <button class="menubtn" type="submit"><img src="image/hamburger.JPG" id ="menu"></button>
```

Comment:
- [ ] You have done incredibly well by using img tag. However, you should ensure that the alt has a text. This is because alt text is critical for those with disabilities because it describes the function and appearance of a photo or graphic they cannot see. Kindly ensure to make your page more accessible by implementing the alt text 👍 .